libnetwork/drivers/bridge/setup_ip_tables_linux_test.go

```diff
@@ -184,3 +203,148 @@ func TestSetupIP6TablesWithHostIPv4(t *testing.T) {
 	createTestBridge(nc, br, t)
 	assertBridgeConfig(nc, br, d, t)
 }
+
+func TestOutgoingNatRules(t *testing.T) {
```

Comment:
```suggestion
func TestOutgoingNATRules(t *testing.T) {
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done.